app/experimenter/base/management/commands/generate-docs.py

```diff
@@ -0,0 +1,51 @@
+import io
+import logging
+import json
+
+from django.core.management import call_command
+from django.template.loader import render_to_string
+from django.core.management.base import BaseCommand
+
+
+logger = logging.getLogger()
+
+
+class Command(BaseCommand):
+    help = "Generates API docs"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--check", default=False, type=bool)
+
+    def handle(self, *args, **options):
+        self.generate_docs(options)
+
+    @staticmethod
+    def generate_docs(options):
+        if options["check"]:
+            output = io.StringIO()
+            call_command("generateschema", "--format=openapi-json", stdout=output)
+            output.seek(0)
+            api_json = output.read()
+
+            with open("/app/experimenter/docs/openapi-schema.json", "r") as f:
```

Comment:
These paths should be relative to project root and probably stored in settings.py